### PR TITLE
Clarified instructions

### PR DESCRIPTION
**Pull request by emersonthis**
I spent all day trying to figure out why I couldn't install my own fork... figured I'd add a note to save someone else the trouble.

### CODE REVIEW

File: doc/05-repositories.md [hunk 1]
```diff
@@ -263,6 +263,8 @@ custom repository has priority over packagist. If you want to rename the
 package, you should do so in the default (often master) branch and not in a
 feature branch, since the package name is taken from the default branch.
 
+Also note that the override will not work if you change the `name` property in your forked repository's composer.json file as this needs to match the original for the override to work.
```
Comment (alcohol) [inline doc/05-repositories.md]: Line length exceeded. Please wrap.

**Comment by alcohol** [comment]
While it is kind of stating the obvious, adding a note might not hurt, since it is still a commonly overlooked mistake.

**Comment by emersonthis** [comment]
It certainly got me. Maybe because I'm new to composer.

**Comment by Seldaek** [comment]
Thanks

**Pull request merged**